dashboard/test/ui/features/blockTrashing.feature

```diff
@@ -0,0 +1,45 @@
+Feature: Blocks can be trashed in certain circumstances
+
+Background:
+  Given I am on "http://learn.code.org/s/course2/stage/19/puzzle/2"
+  # In this stage:
+  # block 18 is the "when run" block
+  # block 19 is an undeletable "set color" block
+  # block 24 is a deletable "turn right" block
```

Comment:
good point, I don't know the blockly code that well

Another option... is it possible to css select based on the block text instead of the id?